lib/src/inbox_message_view.dart

```diff
@@ -8,34 +8,34 @@ import 'package:flutter/widgets.dart';
 
 class InboxMessageView extends StatelessWidget {
   final String messageId;
-  final void Function() onLoadStarted;
-  final void Function() onLoadFinished;
-  final void Function(PlatformException) onLoadError;
-  final void Function() onClose;
+  final void Function()? onLoadStarted;
+  final void Function()? onLoadFinished;
+  final void Function(PlatformException)? onLoadError;
+  final void Function()? onClose;
   static bool hybridComposition = false;
 
   InboxMessageView({
-    @required this.messageId, this.onLoadStarted, this.onLoadFinished, this.onLoadError, this.onClose
+    required this.messageId, this.onLoadStarted, this.onLoadFinished, this.onLoadError, this.onClose
   });
 
   Future<void> onPlatformViewCreated(id) async {
     MethodChannel _channel = new MethodChannel('com.airship.flutter/InboxMessageView_$id');
     _channel.setMethodCallHandler(methodCallHandler);
     _channel.invokeMethod('loadMessage', messageId).catchError( (error) {
-      onLoadError(error);
+      onLoadError!(error);
     });
   }
 
   Future<void> methodCallHandler(MethodCall call) async {
     switch (call.method) {
       case 'onLoadStarted':
-        onLoadStarted();
+        onLoadStarted!();
```

Comment:
same comment as above, I think we may not want to use the null assertion operator here unless I'm missing something.